lib/code_corps/endpoint.ex

```diff
@@ -38,5 +38,6 @@ defmodule CodeCorps.Endpoint do
     key: "_code_corps_key",
     signing_salt: "RJebPcfU"
 
+  plug Corsica, origins: "http://localhost:4200"
```

Comment:
Ha see my other comment. Might be overkill to update a config and then update a test for that config. But then again when a new subdomain is added and it breaks the app we'll know who to blame.